pages/githubandmarkdown.md

```diff
@@ -7,19 +7,22 @@
 ###Prerequisites :
 Already created Username on Github or Set up Github account.
 
-[GitHub](https://help.github.com/categories/writing-on-github/) acts as a source code management system for Open Learning Exchange. It's a very simple open source system that is easy to learn and to use. Please read some of the basic instructions below as a user manual for OLE on GitHub. 
+[GitHub](https://help.github.com/categories/writing-on-github/) acts as a source code management system for Open Learning Exchange. It's a very simple open source system that is easy to learn and to use. Please read some of the basic instructions below as a user manual for OLE on GitHub.
 
-[Markdown Wiki](http://dynalon.github.io/mdwiki/#!quickstart.md) is the software you're currently reading on that OLE uses for our Virtual intern program. Markdown (MD) Wiki allows us to use Markdown and Javascript on GitHub so that it is easily compatible with our current development environment. To be explicit, Markdown is a simplified development language which allows you to format and write text. 
-Below is a screenshot of what Markdown language with a preview looks like: 
+[Markdown Wiki](http://dynalon.github.io/mdwiki/#!quickstart.md) is the software you're currently reading on that OLE uses for our Virtual intern program. Markdown (MD) Wiki allows us to use Markdown and Javascript on GitHub so that it is easily compatible with our current development environment. To be explicit, Markdown is a simplified development language which allows you to format and write text.
+Below is a screenshot of what Markdown language with a preview looks like:
 
 ![Markdown Preview](uploads/images/markdown.png)
 
+## Markdown != HTML
+### Looking at the cheat sheet below, notice that the code on the left is Markdown, and it's HTML equivalent is on the right. 
```

Comment:
This line could be added in parenthesis at the end of line 20. It's good to tell interns, so they don't get confused, but there is no need to overemphasize it.
Also, please change "it's" to "its." ;-)